src/db/error.cr

```diff
@@ -14,10 +14,10 @@ module DB
   # Raised when an established connection is lost
   # probably due to socket/network issues.
   # It is used by the connection pool retry logic.
-  class ConnectionLost < Error
-    getter connection : Connection
+  class ConnectionLost(T) < Error
```

Comment:
Note to future self: check if this is causing any breaking change on db drivers

Reply:
I've checked this against `pg` and it worked fine. I didn't check the other drivers, though.